pkg/sql/logictest/testdata/logic_test/notice

```diff
@@ -23,3 +23,13 @@ SELECT crdb_internal.notice('debug1', 'now you see this'), crdb_internal.notice(
 ----
 DEBUG1: now you see this
 WARNING: and you see this
+
+statement ok
+CREATE DATABASE d;
+CREATE TABLE d.t (x int)
+
+query T noticetrace
+ALTER TABLE d.t RENAME TO d.t2
+----
+NOTICE: renaming tables with a qualification is deprecated
+HINT: use ALTER TABLE ... SET SCHEMA instead
```

Comment:
oh, in this case the correct hint is to `ALTER TABLE t RENAME TO t2` instead.

Reply:
heh, to move x-database you need to dump and restore the tables...

Reply:
I meant to move that error below and change the notice, updated.